hw/sensor/include/sensor/sensor.h

```diff
@@ -482,6 +482,15 @@ struct sensor *sensor_mgr_find_next_bydevname(char *, struct sensor *);
  */
 int sensor_mgr_match_bytype(struct sensor *sensor, void *arg);
 
+/**
+ * Set the sensor poll rate
+ *
+ * @param The devname
+ * @param The poll rate in milli seconds
+ */
+int
+sensor_set_poll_rate_ms(char *devname, uint32_t poll_rate);
```

Comment:
other functions take struct sensor instead of name,  is there a reason this one is special?